src/ontology/edits/oeo-model.omn

```diff
@@ -621,6 +761,22 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1834",
 DataProperty: OEO_00140178
 
     
+DataProperty: OEO_00390080
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2013
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2020",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Has literal value is a data property that links an entity to a type of literal that describes it.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Has literal value is a data property that links an entity to a value of type literal.",
```

Reply:
With making the relevant predicates data properties, the has literal value is not needed at the moment.